src/main/java/core/basesyntax/store/FruitService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.store;
+
+public interface FruitService {
```

Comment:
let's not overcomplicate the code - you don't actually need both `FruitService` and `StorageService` - they both communicate with the storage